src/components/biggive-campaign-card/biggive-campaign-card.tsx

```diff
@@ -130,7 +130,7 @@ export class BiggiveCampaignCard {
               </div>
             ) : null}
 
-            {this.banner !== null ? (
+            {(this.banner !== null && this.banner !== undefined)? (
```

Comment:
I tried doing `typeof this.banner !== 'string'` but it doesn't like that, I get an error that has to be true since its typed as string. This way works though.